protocol/innpv.proto

```diff
@@ -134,6 +138,17 @@ message UtilizationResponse {
   float tensor_utilization = 3;
 }
 
+// DDP message
+// =====================================
+
+message DdpResponse {
+  float forward_time_ms = 1;
+  repeated float bucket_sizes = 2;
+  repeated float expected_max_2gpus = 3;
```

Comment:
This needs a more descriptive name. Also, we should make this general (for >4 GPUs) so that we would not need to change the protobuf later.

Reply:
changed name and made it more general